tests/test_pandas_api.py

```diff
@@ -2029,3 +2029,12 @@ def test_keyed_loc_fixes(q):
         mkt[['k1', 'y']]
     with pytest.raises(KeyError):
         mkt['k1']
+
+def test_values(q):
+    col1 = q('10?100')
+    col2 = q('10?`a`b`c`d')
+    col3 = q('10?`x`y`z`1`2`3')
+    tab = q('{[price; sym; id] ([]p: price; s: sym; i: string id)}', col1, col2, col3)
+    q_table = tab
+    pandas_table = tab.pd()
+    assert pandas_table.values.tolist() == q_table.values.py()

```

Comment:
As you mentioned, the conversion is not perfect. When you execute tab.values.np(), it will not return a np.array but instead a np.array of np.arrays. I believe that, instead of altering the definition of the conversion to numpy or iterating through a list, it is a better option to keep it as it is.